apps/application-system/api/README.md

```diff
@@ -31,3 +31,7 @@ When making changes to the module code, run
 to generate the code needed for openapi and swagger. Then you can visit
 
 `localhost:3333/swagger`
+
+In order to update the graphql schema as well, run
```

Comment:
This command generates a typed fetch client

`yarn nx codegen api 
`
generates the graphql schema